src/coreclr/gc/gc.cpp

```diff
@@ -25392,13 +25420,15 @@ int gc_heap::calculate_new_heap_count ()
     // target_tcp should be configurable.
     float target_tcp = 5.0;
     float target_gen2_tcp = 10.0;
-    float log_base = (float)1.1;
+    float log_base = (float)1.11;
```

Comment:
does that extra decimal have a measurable difference?

Reply:
yes, that's why I changed :) I adjusted the formula for calculating heap growth (so it works for small HCs as well) and needed to adjust this a bit to yield similar results for larger HCs